community/contributing/pr_review_guidelines.rst

```diff
@@ -0,0 +1,351 @@
+    cannot detect.
+
+* **Code only touches the areas announced in the PR (and the commit
+  message).**
+
+  It is always tempting to start fixing random things in code, as you
+  see them. However, this can quickly become a hell to dig through in
+  the commit history. Small touch-ups next to the related area are
+  alright, but often bugs that you can find along the way are better
+  left for their own PRs.
+
+* **Code properly uses Godot's own APIs and patterns.**
+
+  This is true for any reasonably sized project — consistency is very
+  important, and a solution that already exists in the codebase
+  is preferable to an ad-hoc solution.
+
+* **Are core areas affected by the change?**
+
+  Sometimes a PR that is supposed to solve a local problem can have a
+  far-reaching effect, way outside of its scope. If you believe that
+  is the case, make sure to get senior maintainers involved in the
+  process, namely `reduz <https://github.com/reduz>`_ or
+  `vnen <https://github.com/vnen>`_. Their input and approval would
+  be required for changes to the engine's core systems and overall
+  architecture, APIs that touch on all Objects or Nodes, changes
+  to GDScript and GDNative. If the pull request modifies the build
+  system, you can contact `Akien <https://github.com/akien-mga>`_.
+
+5. Iterate with the contributor and improve the PR
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Inevitably, a lot of pull requests would be initially undercooked.
+At this point, you and the contributor should enter a feedback loop
+where they iterate on your notes, recommendations and requests, and
+you look that they have indeed fixed the issues you've highlighted.
+
+Try not to exhaust the contributor with style nitpicks, especially
+if it's still up in the air whether their PR would be accepted at
+all. Preferably, suggestions should come in order of importance:
+first address their overall code design and approach to solving the
+problem, then make sure their code is complying with the engine's
+best practices, and last do the "pretty pass".
+
+Some areas of the engine are more important than others. There is
+a lot of ugly code in the editor and UI components, and while this
+is not ideal, it's not the end of the world either. It's fine to
+suggest improvements there as well, but don't insist too much if
+the end result is achieved and the contribution leaves the code
+reasonably maintainable.
+
+And ultimately, don't feel pressured to deal with the PR all
+alone. Feel free to ask for a helping hand on the `Godot
+Contributors Chat <https://chat.godotengine.org>`_, in the appropriate
+channel or in general. Other teams may already be tagged for review,
+so you can also wait or ask for their assistance.
+
+6. Approve the pull request
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you find that everything is in order, that the PR is addressing
+a problem and does so in the acceptable way (it doesn't have to be
```

Comment:
```suggestion
If you find that everything is in order and that the PR is addressing
a problem in an acceptable way (it doesn't have to be
```

Reply:
This changes the accents in the point that is being made. Why do you want to change it?

Reply:
May have misread it, and reading this part again, it seems that a semicolon would work better since as of right now it reads as if the things "you find" are being listed here (i.e. everything in order, addressing problem, and in an acceptable way).

```suggestion
If you find that everything is in order; that the PR is addressing
a problem and does so in the acceptable way (it doesn't have to be
```

Reply:
Well, there are two things listed here, which are related to "you find":
* that everything is in order;
* that the PR is addressing a problem and does so in the acceptable way.

Is that not clear from my phrasing and punctuation? Would this change help?

```suggestion
If you find that everything is in order, that the PR is addressing
a problem, and that it does so in the acceptable way (it doesn't have to be
```

Reply:
Yeah, that change should work.